src/Stratis.Bitcoin.Features.SmartContracts.Tests/DeterminismValidationTests.cs

```diff
@@ -709,6 +709,36 @@ public void F(float param)
             Assert.Contains(result.Errors, e => e.Message.Contains("System.Single"));
         }
 
+        [Fact]
+        public void Validate_Determinism_MethodParams_Private()
+        {
+            string adjustedSource = @"using System;
+                                            using Stratis.SmartContracts;
+
+                                            public class Test : SmartContract
+                                            {
+                                                public Test(ISmartContractState state)
+                                                    : base(state) {}         
+
+                                                private void F(TestStruct test)
```

Comment:
Can we add public, private, and internal methods here too? And check that the error returned contains the public methodname.

Reply:
👍